src/composables/transactionItemList.ts

```diff
@@ -0,0 +1,128 @@
+import { computed } from '@vue/composition-api';
+import {
+  IActiveMultisigTx,
+  IDefaultComposableOptions,
+  ITokenList,
+  ITransaction,
+  ITx,
+} from '../types';
+import {
+  AETERNITY_CONTRACT_ID,
+  defaultTransactionSortingCallback,
+  getInnerTransaction,
+  TX_FUNCTIONS,
+} from '../popup/utils';
+import { useTransactionTx } from './transactionTx';
+import { usePendingMultisigTransaction } from './pendingMultisigTransaction';
+import { useTransactionAndTokenFilter } from './transactionAndTokenFilter';
+import { i18n } from '../store/plugins/languages';
+
+interface TransactionItemList extends IDefaultComposableOptions {
+  isMultisig?: boolean,
+  token: string,
+}
+
+type ITransactionElement = ITransaction | IActiveMultisigTx
+
+export const useTransactionItemList = ({
```

Comment:
I'd suggest going with the name `useTransactionList` as you are not referring to a single element on the list but to the list as a whole